Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -769,7 +799,11 @@ struct HTTP2FrameDecoder {
     mutating func nextFrame() throws -> (HTTP2Frame, flowControlledLength: Int)? {
         // Start running through our state machine until we run out of bytes or we emit a frame.
         while true {
-            switch (try self.processNextState()) {
+            switch (
+                try self.processNextState(
+                    maximumSequentialContinuationFrames: self.maximumSequentialContinuationFrames
+                )
```

Comment:
We don't need other pass this in here: `maximumSequentialContinuationFrames` is already stored on `self` so we can just read it from `self` in `processNextState`